data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/session/SessionInit.avsc

```diff
@@ -26,6 +26,22 @@
       "type": ["null","string"],
       "doc": "Flow ID of the flow initiating the session. This is only required to inform the Flow Mapper. It should be null when sent to counterparties."
     },
+    {
+      "name": "contextUserProperties",
```

Comment:
So is the idea that the `FlowContextProperties` fills the `contextUserProperties` and then an internal API/access to the map alters `contextPlatformProperties`?

Reply:
Yes exactly that. Platform properties are immutable as far as the user is concerned, they will contain things like "account", list TBD but outside the scope of context propagation implementation. User context is whatever the user wants it to be. Only internal apis can set platform properties, we don't expose the ability to do that to users.